src/go/rpk/pkg/config/params_test.go

```diff
@@ -75,6 +75,32 @@ rpk:
     kafka_api:
         brokers:
             - 127.0.1.1:9647
+`,
+		},
+		{
+			name: "preserve order of admin_api.addresses",
```

Comment:
I'm not sure the purpose of this test -- the input is already ordered, and the change also is ordered, so this is just testing appending. If I change the address ordering below (and keep the change in exp), the test fails.